compiler/rustc_error_messages/locales/en-US/metadata.ftl

```diff
@@ -164,6 +164,7 @@ metadata_failed_write_error =
 
 metadata_missing_native_library =
     could not find native static library `{$libname}`, perhaps an -L flag is missing?
+    .help = only provide the library name `{$suggestedname}`, not the full filename
```

Comment:
```suggestion
    .help = only provide the library name `{$suggestedname}`, without the extension
```

Is this better or worse?

Reply:
I like that it's more explicit about what is wrong, but unix platforms also have a "lib" prefix applied to the filename that the user also needs to remove and saying "without the prefix and extension" seems pretty wordy to me.